plugins/language_haxe.lua

```diff
@@ -54,7 +55,7 @@ syntax.add {
     { pattern = "-?%d+%.[%deE]+",                           type = "number"   },
     { pattern = "-?%d+[%deE]+",                             type = "number"   },
     { pattern = "[%+%-%.=/%*%^%%<>!~|&]",                   type = "operator" },
-    { pattern = "[%a_][%w_]*%s*%f[(]",                      type = "function" },
+    { pattern = "[%a_][%w_]*%f[(]",                         type = "function" },
```

Comment:
ahh, my bad I gave you an incorrect pattern before:

wrong one: 
```lua
{ pattern = "[%a_][%w_]()*%s*%f[(]",   type = {"function", "normal"} },
```
good one:
```lua
{ pattern = "[%a_][%w_]*()%s*%f[(]",   type = {"function", "normal"} },
```
Sorry that I made the mistake when adjusting the pattern and left the asterisk as part of the second statement match :sweat: the good one should work:

![20230216-121941](https://user-images.githubusercontent.com/1702572/219425576-1bbe11d1-fedd-4dd4-99a3-45e255251d2d.png)

Reply:
Done!

Reply:
Thanks a lot!